### PR TITLE
Change placement of cpmoverride for Rubicon

### PR DESCRIPTION
**Pull request by DGarbar**
Change placement of cpm**o**verride to 
`ext.**prebid.bidders**.rubicon.debug.cpmoverride`
`imp[].ext.prebid.rubicon.debug.cpmoverride`

### CODE REVIEW

File: adapters/rubicon/rubicon_test.go [hunk 1]
```diff
@@ -1490,10 +1492,12 @@ func TestOpenRTBResponseOverridePriceFromCorrespondingImp(t *testing.T) {
 				}
```
Comment (guscarreon) [inline adapters/rubicon/rubicon_test.go:1492]: `cpmOverride` will also need to be modified I think

File: adapters/rubicon/rubicon_test.go [hunk 2]
```diff
@@ -1490,10 +1492,12 @@ func TestOpenRTBResponseOverridePriceFromCorrespondingImp(t *testing.T) {
...
 		}},
-		Ext: json.RawMessage(`{"rubicon": {
-				"debug": {
-					"cpmOverride" : 10 
-				}}}`),
+		Ext: json.RawMessage(`{"prebid": {
+			"bidders": {
+				"rubicon": {
+					"debug": {
+						"cpmOverride": 10
```
[the remark this answers is not in the document]
Comment (DGarbar) [reply adapters/rubicon/rubicon_test.go]: Thanks, good point. 
Fixed.
But I wonder, why this case insensitive ?

[the remark this answers is not in the document]
Comment (guscarreon) [reply adapters/rubicon/rubicon_test.go]: I know no tests were failing, I just think it is better for consistency

[the remark this answers is not in the document]
Comment (DGarbar) [reply adapters/rubicon/rubicon_test.go]: I completely agree with that, but is it normal, that tests are case insensitive.
Bc this test also valid for `cPMoVeRriDe`

**Pull request merged**